tests/views_integration/Tribe/Events/Views/V2/Partials/Components/__snapshots__/Ical_LinkTest__test_render_with_gcal_only__1.php

```diff
@@ -1,12 +1,12 @@
 <?php return '<div class="tribe-events-c-ical tribe-common-b2 tribe-common-b3--min-medium">
 	<a
 		class="tribe-events-c-ical__link"
-		title="Subscribe via Google Calendar"
+		title=""
```

Comment:
Are these updated snapshots correct? It appears the title is missing and there is no text in the label.

Reply:
Oh, nice catch, and weird.

Reply:
Fixed!